migrate.go

```diff
@@ -50,25 +51,36 @@ func runMigrations(conf *DBConf, migrationsDir string, target int64) {
     current, e := ensureDBVersion(db)
     if e != nil {
         log.Fatalf("couldn't get DB version: %v", e)
-    }
+		}
 
-    mm, err := collectMigrations(migrationsDir, current, target)
-    if err != nil {
-        log.Fatal(err)
+    var mm *MigrationMap
+    if allowOutOfOrder {
+        mm, err = collectAllUnappliedMigrations(migrationsDir, current, target, db)
+        if err != nil {
```

Comment:
It's cleaner if the error check step lives after the if/else, since it's the same in both cases